src/core/Akka/Actor/Internals/ActorSystemImpl.cs

```diff
@@ -127,7 +127,8 @@ public override ActorSelection ActorSelection(string actorPath)
 
         private void ConfigureScheduler()
         {
-            _scheduler = new DedicatedThreadScheduler(this);;// new TaskBasedScheduler();
+            var schedulerType = Type.GetType(_settings.SchedulerClass, true);
```

Comment:
throw/log if scheduler type resolves to null as this would break the entire system